src/components/Trade/TradeInput.tsx

```diff
@@ -224,7 +252,7 @@ export const TradeInput = ({ history }: RouterProps) => {
                     isNumericString={true}
                     onValueChange={e => {
                       onChange(e.value)
-                      if (e.value !== value) {
+                      if (e.value !== value && sellTradeAsset?.asset && buyTradeAsset?.asset) {
```

Comment:
This `sellTradeAsset?.asset && buyTradeAsset?.asset` expression is present multiple times across the component, do we want to abstract it away?